### ISSUE TITLE
`cargo crev` ID list

### ISSUE THREAD

**Issue by Lokathor** [rust-secure-code/safety-dance#31]
We might care to store a list of `cargo crev` URLs in a markdown file so that people can discover IDs more easily.

We don't even need to specifically endorse one user or another, people can decide for themselves who to trust, we just need to make it more visible.

**Comment by danielhenrymantilla** [comment]
Agreed. I haven't set up `cargo crev` for myself yet, but I know I would greatly appreciate having a list of ids, especially for the people in safety dance.

**Comment by Shnatsel** [comment]
Discover what reviewers are even available is definitely needed, but I feel this is more of a work item for cargo-crev itself than for safety-dance. Please open an issue at https://github.com/crev-dev/cargo-crev/issues

**Comment by Shnatsel** [comment]
Hot take: just put your crev IDs here. This will provide the discoverability if anyone is actively looking, and will not look _too_ much like an endorsement.

**Comment by tarcieri** [comment]
Yes please, document them *somewhere* (this thread seems like a great place) and then we can figure out how to do a better integration.

**Comment by Lokathor** [comment]
Post the git urls for your crev info here and I'll edit them into the opening post as I have time.

**Comment by dpc** [comment]
We have a list of active proof repositories here: https://github.com/crev-dev/cargo-crev/wiki/List-of-Proof-Repositories . Also - as long as you cross-trust your IDs, any person that trust any of you, will quite easily download reviews from all others with `cargo crev repo fetch trusted`.
